toolbox/entitlement/entitle.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash -e
+
+extract_entitlement_key() {
+    resource=$1
+    key=$2
+    RESOURCE_NAME=50-entitlement-pem
+    cat "$resource" \
+        | yq . \
+        | jq -r -c 'select(.metadata.name=="'${RESOURCE_NAME}'")|.spec.config.storage.files[0].contents.source' \
+        | sed 's|data:text/plain;charset=utf-8;base64,||g' \
+        | base64 -d \
+        > "$key"
+}
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../_common.sh
```

Comment:
hum, I missed that @dagrayvid 

```
source ${THIS_DIR}/../_common.sh
...
toolbox/.../....sh
```

this doesn't work well, because `_common.sh` sets ansible log variable, so they won't be re-set inside the toolbox script, and the logs will overwrite each others:

see the entitlement log dirs from [there](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/117/pull-ci-openshift-psap-ci-artifacts-master-gpu-operator-e2e/1389287582428303360/artifacts/gpu-operator-e2e/presubmit-operatorhub/artifacts/)

That's the issue I had with [`must-gather.sh`](https://github.com/openshift-psap/ci-artifacts/blob/master/toolbox/gpu-operator/must-gather.sh#L9)

could you try to fix it, either with a "hack" as in `must-gather.sh`, or with a cleaner solution if you can find one :)